woocommerce/woocommerce
# Use nbsp to separate * denoting required fields.

fixes #19569 

Testing instructions:
1. When logged out, navigate to My Account (`.../my-account/`)
Check space/nbsp for following example fields:
Username or email address *
Password *

2. Navigate to Single product view-> Reviews:
Check space/nbsp for following review form fields:
Your review *
Name * 
Email *

3. When logged in, navigate to My account/Account details (`.../my-account/edit-account/`):
Check space/nbsp for following example fields:
First name *
Last name *
Display name *
Email address *

4. Enable the Terms & conditions in Woo settings > Checkout, then put something into the cart, proceed to checkout. 
Check space/nbsp for billing fields:
Most of the those should be marked with a * 
Also, terms and conditions marked with * should have correct space type now.
